prow/cmd/README.md

```diff
@@ -39,9 +48,30 @@ These are small tools that are automatically added to ProwJob pods for jobs that
 
 * [`clonerefs`](/prow/cmd/clonerefs)
 * [`initupload`](/prow/cmd/initupload)
+* [`gcsupload`](/prow/cmd/gcsupload)
 * [`entrypoint`](/prow/cmd/entrypoint)
 * [`sidecar`](/prow/cmd/sidecar)
 
 ## Base Images
 
 The container images in [`images`](/images) are used as base images for Prow components.
+
+## TODO: undocumented
+
+* [`admission`](/prow/cmd/admission)
+* [`grandmatriarch`](/prow/cmd/grandmatriarch)
+* [`pipeline`](/prow/cmd/pipeline)
+* [`tackle`](/prow/cmd/tackle)
+
+## Deprecated
+
+* [`autobump`](/prow/cmd/autobump) autobumps a prow installation. Deprecated because you should use `generic-autobumper` instead
+* [`cm2kc`](/prow/cmd/cm2kc) is a CLI tool used to convert a [clustermap file][clustermap docs] to a [kubeconfig file][kubeconfig docs]. Deprecated because we have moved away from clustermaps; you should use `gencred` to generate a [kubeconfig file] directly.
+
+[github branch protection]: https://help.github.com/articles/about-protected-branches/
+[clustermap docs]: https://github.com/kubernetes/test-infra/blob/1c7d9a4ae0f2ae1e0c11d8357f47163d18521b84/prow/getting_started_deploy.md#run-test-pods-in-different-clusters
+[kubeconfig docs]: https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/
+[`gencred`]: /gencred
+[gerrit]: https://www.gerritcodereview.com/
+[`updateconfig` Prow plugin]: /prow/plugins/updateconfig
```

Comment:
This is deprecated?

Reply:
this section is reference section for hyperlinks, this is a markdown trick, this line means any where you use [`updateconfig` Prow plugin] it means hyperlinked to `/prow/plugins/updateconfig`

Reply:
What is used instead?

Reply:
Some of these are not used anywhere, maybe not needed?

Reply:
Sorry I guess I was interpreting @alvaroaleman's statement to mean "oh we don't use `updateconfig` anymore" and I was trying to ask "what do we use instead of `updateconfig` to update prow's config now?"

I'll double check these links, I added them all, maybe I missed linking back to them

Reply:
Added a `<!-- links -->` comment to make it clear these are links, not part of the deprecated section

Also caught that I hadn't linked gerrit or gencred above, added